src/main/kotlin/com/dcd/server/core/domain/application/service/impl/CreateDockerFileServiceImpl.kt

```
@@ -81,13 +85,14 @@ class CreateDockerFileServiceImpl(
             )
 
         try {
-            if (!file.exists())
-                file.createNewFile()
-            file.writeText(fileContent)
+            fileOperationPort.writeFile(Paths.get("${applicationPath}", "Dockerfile"), fileContent)
         } catch (e: IOException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)